spec/rspec/support/ruby_features_spec.rb

```diff
@@ -123,6 +123,10 @@ def ripper_works_correctly?
             ripper_can_parse_source_including_keywordish_symbol?
         end
 
+        def self.ripper_supported?
+          RubyFeatures.ripper_supported?
```

Comment:
This is the subject under test, its not a valid way to see if ripper is supported, `defined?(Ripper)` would be ok.